shelley/chain-and-ledger/executable-spec/test/Test/Shelley/Spec/Ledger/NonTraceProperties/Validity.hs

```diff
@@ -56,7 +56,9 @@ data ValidationError
   | -- | The transaction fee is too small
     FeeTooSmall Coin Coin
   | -- | Value is not conserved
-    ValueNotConserved Coin Coin
+    ValueNotConserved
```

Comment:
Why change this? that seemed useful

Reply:
Value is parametrized by crypto. This means this whole type must be parametrized also. I instead made the error 
`ValueNotConserved ByteString ByteString` , which shows the Values as bytestrings

Reply:
I think you probably *should* just pipe the parameter through everywhere, although I can see that's annoying.